### PR TITLE
safe: add page

### PR DESCRIPTION
**Pull request by Aracki**
I've added [safe](https://github.com/starkandwayne/safe) - a Vault CLI on steroids.

### CODE REVIEW

File: pages/common/safe.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`safe env`
+
+- Provide a tree hierarchy listing of all reachable keys in the Vault:
+
+`safe tree {{path}}`
+
+- Move a secret from one path to another:
+
+`safe move {{old_path/secret}} {{new_path/secret}}`
```
Comment (schneiderl) [inline pages/common/safe.md]: I don't like these parameter names very much. I would prefer if we could use the default style like `path/to/secret` but I don't know how to rewrite this properly so I think that's good haha.

File: pages/common/safe.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Generate a new 2048-bit SSH keypair and store it in vault:
+
+`safe ssh 2048 {{path/secret}}`
```
Comment (schneiderl) [inline pages/common/safe.md]:
```suggestion
`safe ssh 2048 {{path/to/secret}}`
```

File: pages/common/safe.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Set non-sensitive keys for a secret:
+
+`safe set {{path/secret}} {{key}}={{value}}`
```
Comment (schneiderl) [inline pages/common/safe.md]:
```suggestion
`safe set {{path/to/secret}} {{key}}={{value}}`
```

File: pages/common/safe.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- Set auto-generated password in a secret:
+
+`safe gen {{path/secret}} {{key}}`
```
Comment (schneiderl) [inline pages/common/safe.md]:
```suggestion
`safe gen {{path/to/secret}} {{key}}`
```

File: pages/common/safe.md (hunk 2 above)
Comment (agnivade) [inline pages/common/safe.md]: 2048 in tokens (`{{}}`)

File: pages/common/safe.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Authenticate the CLI client against the Vault server, using an authentication token:
+
+`safe auth {{authentication_token}}`
+
+- Print the environment variables describing the current target:
+
+`safe env`
+
+- Provide a tree hierarchy listing of all reachable keys in the Vault:
```
Comment (agnivade) [inline pages/common/safe.md]: Display a tree hierarchy of all reachable keys for a given path:

File: pages/common/safe.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`safe tree {{path}}`
+
+- Move a secret from one path to another:
+
+`safe move {{old_path/secret}} {{new_path/secret}}`
+
+- Generate a new 2048-bit SSH keypair and store it in vault:
```
Comment (agnivade) [inline pages/common/safe.md]: Remove "in vault"

File: pages/common/safe.md (hunk 1 above)
Comment (Aracki) [reply pages/common/safe.md]: @schneiderl Thanks for the review. Maybe to put `old/path/secret` `new/path/secret`. Not sure  🤔

**Comment by Aracki** [comment]
Thanks @agnivade & @schneiderl for the reviews. I've done some fixes.

**Pull request merged**